packages/paste-website/src/pages/components/checkbox/index.mdx

```diff
@@ -101,6 +106,22 @@ Use a checkbox to present a user with a binary (e.g. on/off) choice that is part
 - When in an error state display an inline error message below the offending checkbox that clearly describes the error along with an icon that depicts the severity.
 - When displaying additional content based on checking a checkbox, be sure that the new content appears after the checkbox so that it is naturally discoverable by assistive technology users.
 
+## Controlled vs. uncontrolled checkboxes
+
+The checkbox can either be controlled, meaning there is an external state that determines if the checkbox is checked or not, or uncontrolled, meaning the checkbox manages its own state.
+
```

Comment:
Super helpful to have this context in the docs, might be worth adding a sentence about why one would want an uncontrolled vs controlled checkbox